### ISSUE TITLE
Exception when running memories:video-setup

### ISSUE THREAD

**Issue by QSaman** [pulsejet/memories#233]
Hello,

I'm running nextcloud in Odroid N2 (Arm based). I don't want to do the trans-coding. Whenever I play a video, I see "video not configured. Run occ memories:video-setup". When I run that comment I see this error:

```
ffmpeg is installed
ffprobe is installed
Checking for go-vod binary
Trying go-vod from 
An unhandled exception has been thrown:
TypeError: chmod(): Argument #1 ($filename) must be of type string, bool given in /var/www/html/custom_apps/memories/lib/Command/VideoSetup.php:108
Stack trace:
#0 /var/www/html/custom_apps/memories/lib/Command/VideoSetup.php(108): chmod(false, 493)
#1 /var/www/html/3rdparty/symfony/console/Command/Command.php(255): OCA\Memories\Command\VideoSetup->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /var/www/html/3rdparty/symfony/console/Application.php(1009): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /var/www/html/3rdparty/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand(Object(OCA\Memories\Command\VideoSetup), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /var/www/html/3rdparty/symfony/console/Application.php(149): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /var/www/html/lib/private/Console/Application.php(213): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 /var/www/html/console.php(100): OC\Console\Application->run()
#7 /var/www/html/occ(11): require_once('/var/www/html/c...')
```

Is there an easy way to just play the original videos without trans-coding?

**Comment by pulsejet** [comment]
That's weird. You just need to add this to `config.php` to disable transcoding:
```php
'memories.no_transcode' => true,
```

**Comment by d89don** [comment]
I've been seeing basically the same error.

```
ffmpeg is installed
ffprobe is installed
Checking for go-vod binary
Trying go-vod from /config/www/nextcloud/apps/memories/exiftool-bin/go-vod-amd64
go-vod is installed!

You can use transcoding and HLS streaming
This is recommended for better performance, but has implications if
you are using external storage or run Nextcloud on a slow system.

Read the following documentation carefully before continuing:
https://github.com/pulsejet/memories/wiki/Configuration

Do you want to enable transcoding and HLS? [Y/n]
An unhandled exception has been thrown:
TypeError: trim(): Argument #1 ($string) must be of type string, bool given in /config/www/nextcloud/apps/memories/lib/Command/VideoSetup.php:118
Stack trace:
#0 /config/www/nextcloud/apps/memories/lib/Command/VideoSetup.php(118): trim()
#1 /config/www/nextcloud/3rdparty/symfony/console/Command/Command.php(255): OCA\Memories\Command\VideoSetup->execute()
#2 /config/www/nextcloud/3rdparty/symfony/console/Application.php(1009): Symfony\Component\Console\Command\Command->run()
#3 /config/www/nextcloud/3rdparty/symfony/console/Application.php(273): Symfony\Component\Console\Application->doRunCommand()
#4 /config/www/nextcloud/3rdparty/symfony/console/Application.php(149): Symfony\Component\Console\Application->doRun()
#5 /config/www/nextcloud/lib/private/Console/Application.php(213): Symfony\Component\Console\Application->run()
#6 /config/www/nextcloud/console.php(100): OC\Console\Application->run()
#7 /config/www/nextcloud/occ(11): require_once('...')

```
Running Nextcloud Hub 3 (25.0.1) on Docker version 20.10.17, build 100c701, Memories

**Comment by pulsejet** [comment]
> I've been seeing basically the same error.

@d89don try typing a `y` instead of just pressing enter.

**Comment by d89don** [comment]
Thanks for the quick response. I don't get a chance to type anything, the prompt and error happen simultaneously. I've tried removing the addon and starting over but the same thing is happening.

**Comment by pulsejet** [comment]
How did you run the command?

**Comment by d89don** [comment]
I’m running it from Linux CLI on Ubuntu 22.04.1 LTS. I’m not using any additional arguments, just as presented in the documentation.

**Comment by pulsejet** [comment]
You mentioned you're running in docker. Did you exec into docker to run the command? E.g. `docker exec <container> php occ ...` won't work. You need `-it`

**Comment by d89don** [comment]
Yes that was exactly what I was using! `-it` worked first try. Thanks brother

**Comment by pulsejet** [comment]
This is fixed in 4.8.2
